docs/csharp/language-reference/builtin-types/bool.md

```diff
@@ -2,8 +2,12 @@
 title: "bool type - C# reference"
 ms.date: 11/26/2019
 f1_keywords: 
-  - "bool"
-  - "bool_CSharpKeyword"
+  - bool
+  - bool_CSharpKeyword
+  - "true"
+  - "false"
+  - true_CSharpKeyword
+  - false_CSharpKeyword
```

Comment:
Just curious, is there some sort of format or syntax to these keywords? Why is true_CSharpKeyword special? I see all sorts of other ones too.

Reply:
I'm not sure why it's special, I'm basically just following a spreadsheet that said which keywords were missing, I think based on keywords that existed in the old MSDN topic and keywords that users pressed F1 on. But I'm not 100% sure.

Reply:
They're defined in Roslyn repository AFAIK.

Reply:
https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs